api_public.yaml

```diff
@@ -33,9 +34,12 @@ definitions:
         type: string
       minAppVersion:
         type: string
+      minAppVersionPagoPa:
```

Comment:
why aren't these snake_cased as all the other APIs props ?

Reply:
Perhaps due to a mistake while introducing the `minAppVersion` property. Now is it better to use the snake format for `minAppVersionPagoPa` and have an API that returns mixed format for properties name? The `minAppVersion` property cannot be changed for compatibility issues.

Reply:
I'll check if `minAppVersion` is really used into the production app

Reply:
The property `minAppVersion` is not used in the app yet. The PR teamdigitale/io-app#1377 that could use it is not yet merged. I'll do the refactoring.

Reply:
Fixed into f632158